packages/webview_flutter/webview_flutter/pubspec.yaml

```diff
@@ -19,9 +19,18 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  webview_flutter_android: ^2.8.0
-  webview_flutter_platform_interface: ^1.9.3
-  webview_flutter_wkwebview: ^2.7.0
+  # FOR TESTING ONLY. DO NOT MERGE.
```

Comment:
FWIW if you use `dependency_overrides` to do the path-based deps, the analyzer won't fail in CI.

Reply:
changed to use `dependency_overrides`

Reply:
@stuartmorgan this seems to be causing problems where it is not finding the overridden version of these packages.

Reply:
Ah right, you'll need to add the overrides in the examples as well. (Filed https://github.com/flutter/flutter/issues/111091 for the tooling not handling that correctly.)

Reply:
Nice that sorted it